src/Nest/Document/Single/Get/GetResponse.cs

```diff
@@ -33,6 +33,12 @@ public interface IGetResponse<out TDocument> : IResponse where TDocument : class
 
 		[JsonProperty("_version")]
 		long Version { get; }
+
+		[JsonProperty("_primary_term")]

```

Comment:
should this be `long?` on `6.x` ? Not returned on older versions.

Reply:
I've looked into this, there are other areas of the code that have implemented both properties as `long` and not `long?`...

Reply:
They are implemented as `long` in places where a value is always returned, and as nullable in places where a value is returned only if a request parameter is provided that indicates to return it